pennylane/gradients/pulse_gradient.py

```diff
@@ -40,11 +39,13 @@
 has_jax = True
 try:
     import jax
+
+    jnp = jax.numpy
```

Comment:
Is that the same the same as `import jax.numpy as jnp` ?

Reply:
Yup, as far as I know. At least after having imported JAX already, I presume :thinking:

Reply:
FYI: https://stackoverflow.com/questions/31716388/python-difference-between-import-as-vs-variable-assignment
In effect, the things are the same up to minor leaks into the  local namespace (which is irrelevant because we have proper naming in PL).
However, there seem to be cases where `import jax.numpy as jnp` is more accessible in some IDE(s), so I'm going to change this to the explicit import :)